test_flatten.py

```
@@ -47,12 +59,56 @@ def test_flatten_skip_extension(source_dir, dest_dir, log_file_path):
+    monkeypatch.delenv("TARGET_EXTENSIONS", raising=False)
+    
+    # Setup: empty source directory (or only ignored files)
+    (source_dir / "file.txt").touch()
+    
+    # Execute with default extensions (txt is ignored)
+    flatten_directory_files(source_dir, dest_dir, logger=test_logger)
+    
+    # Verify info log
+    assert "対象ファイルが見つかりませんでした" in caplog.text
 
+def test_flatten_copy_error(source_dir, dest_dir, monkeypatch, caplog, test_logger, log_file_path):
```

Ruff (0.14.7)

Unused function argument: `log_file_path`

(ARG001)